ros/kxreus/euslisp/kxr-interface.l

```diff
@@ -65,7 +65,12 @@
                                     (format nil "/~A/stretch_interface" controller-name))
                                   kxr_controller::StretchAction
                                   :groupname groupname))
-   (dolist (action (list servo-on-off-client stretch-client))
+   (setq pressure-control-client (instance ros::simple-action-client :init
+                                  (if namespace (format nil "/~A/~A/pressure_control_interface" namespace controller-name)
+                                    (format nil "/~A/pressure_control_interface" controller-name))
+                                  kxr_controller::PressureControlAction
+                                  :groupname groupname))
+   (dolist (action (list servo-on-off-client stretch-client pressure-control-client))
```

Comment:
The pressure-control-client is not supported on the rcb4 board, so it is better not to check it.